Libraries/Image/RCTImageView.m

```diff
@@ -44,6 +44,8 @@ @interface RCTImageView ()
 @property (nonatomic, copy) RCTDirectEventBlock onLoad;
 @property (nonatomic, copy) RCTDirectEventBlock onLoadEnd;
 
+@property (nonatomic, assign, getter=isDisplayingSource) BOOL displayingSource;
```

Comment:
I think this can just be an internal ivar

    BOOL _displayingSource;

Unless there's some reason why it needs to be part of the public interface?